apps/cmp.c

```diff
@@ -2047,6 +2050,22 @@ static int save_free_certs(OSSL_CMP_CTX *ctx,
     return n;
 }
 
+static int save_cert(OSSL_CMP_CTX *ctx, X509 *cert,
+                     const char *destFile, const char *desc)
+{
+    STACK_OF(X509) *certs ;
```

Comment:
Nit: extra space before `;`

Reply:
Oops, fixed,
as well as there further formatting nits (missing empty line after local decls) in the same file detected by `check-format.pl`.